packages/core/src/interceptor/types.ts

```diff
@@ -17,24 +18,24 @@ export interface InterceptorBagDetails {
  * @typeParam Context - The type of the context that will be injected
  * @typeParam State - The type of the optional state that can be used to propagate state between interceptors
  */
-export type InterceptorBag<IBD extends InterceptorBagDetails = InterceptorBagDetails, AdditionalProps = {}> = {
+export type InterceptorBag<IBD extends InterceptorBagGenerics> = {
```

Comment:
the InterceptorBag interface has been simplified.
It now accepts 1 generic object interface, that contains all the defined generics.